error-handling/src/main/java/com/palantir/remoting2/errors/SerializableStackTraceElement.java

```diff
@@ -30,13 +30,23 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public abstract class SerializableStackTraceElement implements Serializable {
 
-    public abstract Optional<String> getClassName();
+    public abstract String getClassName();
```

Comment:
Not sure if we really needed these `Optionals` as `StackTraceElement` has only the following constructor-
```
    public StackTraceElement(String declaringClass, String methodName,
                             String fileName, int lineNumber) {
        this.declaringClass = Objects.requireNonNull(declaringClass, "Declaring class is null");
        this.methodName     = Objects.requireNonNull(methodName, "Method name is null");
        this.fileName       = fileName;
        this.lineNumber     = lineNumber;
    }
```
And given that we're 2.0-ing it seems like a good opportunity to make this breaking change.